cartridges/app_storefront_custom2/cartridge/static/default/css/experience/components/commerceAssets/instagramBanner.css

```diff
@@ -0,0 +1,32 @@
+
+.banner{
```

Comment:
Even if you will only load the css in one file, it is a good practice to use some more specific css selectors, so that you make sure, that these styles won't be applied globally or affect some other iframes.

Reply:
fixed